<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_14">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_15">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dest">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionScores"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScores">
   <property name="text">
    <string>Scores</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_source</tabstop>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_10</tabstop>
  <tabstop>btn_11</tabstop>
  <tabstop>btn_12</tabstop>
  <tabstop>btn_13</tabstop>
  <tabstop>btn_14</tabstop>
  <tabstop>btn_15</tabstop>
  <tabstop>btn_dest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
